src/gameObjects/map_arcade/jessie.gameObject.js

```diff
@@ -0,0 +1,34 @@
+import { scaleFactor } from '../../constants';
+
+export const jessie = (k, map, spawnpoints) => {
+    k.loadSprite('jessie', './characters.png', {
```

Comment:
the path to the assets has changed it is ow `./assets/sprites/characters.png`
Can you fix this please :pray: ?